target/linux/rb532/image/Makefile

```diff
@@ -36,9 +36,11 @@ define Build/lzma-loader-elf
 endef
 
 define Device/nand
+  DEVICE_VENDOR := MikroTik
+  DEVICE_MODEL := RouterBOARD 532
+  DEVICE_VARIANT := NAND
```

Comment:
From what I can see on [WikiDevi](https://wikidevi.com/wiki/MikroTik_RouterBOARD_532_(RB532)) this board always got NAND flash for storing the OS. Hence I'd drop the DEVICE_VARIANT as there are no other variants.

Reply:
ack

Reply:
Fine with me, too.